diff/testdata/sample_onlyin_line_isnt_a_file_header.diff

```diff
@@ -0,0 +1,13 @@
+diff -u source_a/file_1.txt  source_b/file_1.txt
+--- source_a/file_1.txt
++++ source_b/file_1.txt
+@@ -2,3 +3,4 @@
+ To be, or not to be, that is the question:
+-Whether 'tis nobler in the mind to suffer
++The slings and arrows of outrageous fortune,
++Or to take arms against a sea of troubles
+ And by opposing end them. To die—to sleep,
+Only in universe!
+Only in source_a: file_2.txt
+Only in source_b: file_3.txt some unrelated stuff here.
```

Comment:
when does "some unrelated stuff" happen? That runs counter to my suggestion above around just splitting on `: `, which means a more restrictive regex makes sense.

Reply:
Just example that there can be smth in the same line after `Only in` message. But considering not strict check, this `some unrelated stuff here` will be a part of filename.